projects/aca-folder-rules/src/lib/rule-details/conditions/rule-simple-condition.ui-component.html

```diff
@@ -22,6 +22,14 @@
   </mat-form-field>
 
   <mat-form-field class="aca-rule-simple-condition__form__parameter-input">
-    <input matInput placeholder="{{ 'ACA_FOLDER_RULES.RULE_DETAILS.PLACEHOLDER.VALUE' | translate }}" type="text" formControlName="parameter" data-automation-id="value-input">
+    <mat-select formControlName="parameter" data-automation-id="value-select" *ngIf="selectedField?.name === 'mimetype'; else valueInput">
```

Comment:
do not add automation ID if you are not adding an e2e test to use it... also, the id value should be unique across the application

Reply:
removed automation-id